source/fe/mapping_fe_field.cc

```diff
@@ -568,9 +548,9 @@ MappingFE<dim,spacedim,DH,VECTOR>::fill_fe_values (
 
```

Comment:
Yes, I have that too. It would be worth fixing as it's a bit annoying. I think the correct data type for the variable is something like
```
  Tensor<2,dim> result[spacedim];
```
This will automatically initialize it.